motors/dc_motor_class.c

```diff
@@ -112,7 +112,7 @@
  * .
  * `time_sp`
  * : (read/write) Sets the time setpoint used with the `run-timed` command.
- *   Units are in milliseconds.
+ *   Units are in milliseconds. Values must not be negative.
```

Comment:
does writing a negative number to this attribute return `-EINVAL`? If not, it probably should.